### PR TITLE
Feature/AA-5/Db init

### PR DESCRIPTION
**Pull request by Z4K1RainbowDash**
- Added db models
- Added repositories
- Set up database connection and migrations

### CODE REVIEW

**Comment by adamB-prog** [comment]
image
This line throws an InvalidOperationException. Isn't it supposed to be before builder.Build()

File: AppAbyss.WebApi/Program.cs [hunk 1]
```diff
@@ -7,13 +12,21 @@
...
+});
+
 var app = builder.Build();
 
 // Configure the HTTP request pipeline.
 if (app.Environment.IsDevelopment())
 {
     app.UseSwagger();
     app.UseSwaggerUI();
+    builder.Services.AddDatabaseDeveloperPageExceptionFilter();
```
Comment (adamB-prog) [inline AppAbyss.WebApi/Program.cs]: This line throws an InvalidOperationException.

**Review by adamB-prog — CHANGES_REQUESTED** [review]

File: AppAbyss.WebApi/Program.cs (hunk 1 above)
Comment (Z4K1RainbowDash) [reply AppAbyss.WebApi/Program.cs]: I have fixed it.

**Review by adamB-prog — APPROVED** [review]
Thank you for the changes!

**Pull request merged**